docs/data-sources/packer_image.md

```diff
@@ -24,25 +26,52 @@ data "hcp_packer_image" "foo" {
   region         = "us-east-1"
 }
 
-output "packer-registry-ubuntu" {
+data "hcp_packer_image" "bar" {
+  bucket_name    = "hardened-ubuntu-16-04"
+  cloud_provider = "aws"
+  iteration_id   = data.hcp_packer_iteration.hardened-source.ulid
+  region         = "us-west-1"
+}
+
+output "packer-registry-ubuntu-east-1" {
   value = data.hcp_packer_image.foo.cloud_image_id
 }
+
+output "packer-registry-ubuntu-west-1" {
+  value = data.hcp_packer_image.bar.cloud_image_id
+}
+
+# Alternatively if sourcing a single image
+
+data "hcp_packer_image" "baz" {
+  bucket_name    = "hardened-ubuntu-16-04"
+  cloud_provider = "aws"
+  channel        = "production"
+  region         = "us-east-1"
+}
+
+output "packer-registry-ubuntu-east-1-alt" {
+  value = data.hcp_packer_image.baz.cloud_image_id
+}
```

Comment:
```suggestion

### Alternatively, if sourcing a single image

data "hcp_packer_image" "baz" {
  bucket_name    = "hardened-ubuntu-16-04"
  cloud_provider = "aws"
  channel        = "production"
  region         = "us-east-1"
}

output "packer-registry-ubuntu-east-1-alt" {
  value = data.hcp_packer_image.baz.cloud_image_id
}
```

I'm having a hard time adding the code block because the markdown is getting confused with this comment markdown. I'm suggesting moving the alternative into a separated sub-section and keeping both examples separated because all together looks big and hard to quickly see how it works.